packages/@okta/courage-for-signin-widget/webpack.config.js

```diff
@@ -1,11 +1,12 @@
 /* global module __dirname */
-const path = require('path');
-const packageJson = require('./package.json');
-const EMPTY = path.resolve(__dirname, 'src/empty');
-const SHARED_JS = path.resolve(__dirname, 'node_modules/@okta/courage/src');
+const { resolve } = require('path');
+const PACKAGE_JSON = require('./package.json');
+const EMPTY = resolve(__dirname, 'src/empty');
+const SHARED_JS = resolve(__dirname, 'node_modules/@okta/courage/src');
 const { BundleAnalyzerPlugin } = require('webpack-bundle-analyzer');
 const { BannerPlugin } = require('webpack');
-const DIST_FILE_NAME = 'courage-for-signin-widget';
+const PUBLISH_DIR = resolve(__dirname, '../courage-dist');
```

Comment:
nit: May consider moving `courage-dist` to a constant (just in case we ever re-use or change the value)

Reply:
wouldn't worry for now until it's going to be used in another place.